.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -114,9 +114,15 @@ steps:
         artifactName: 'vhd-release-notes-${{ parameters.artifactName }}'
         targetPath: 'release-notes.txt'
 
+# Use jq to reformat the image-bom.json file
+  - bash: |
```

Comment:
Can we add some logs to show this step in the build logs?

Reply:
Add the display name.